### PR TITLE
[FE-12] User Interface for Posts

### PR DESCRIPTION
**Pull request by ogi02**
User interface for posts:
- create posts form for creators
- display posts to high enough tiers of donation
- like and unlike option
- display patches of 10 posts with 'load more' button

### CODE REVIEW

File: odyssey/client/src/Profiles/UserProfile.svelte [hunk 1]
```diff
@@ -74,6 +85,56 @@
...
+		return temp;
+	}
+
+	async function likePost(post_id) {
+		const response = await fetchPost('http://localhost:3000/likePost', {
+			post_id: post_id
+		});
+
+		posts.find(post => post._id.$oid === post_id).isLiked = true;
+		posts = posts;
```
Comment (GabrielaY) [inline odyssey/client/src/Profiles/UserProfile.svelte:114]: Hi! Can you explain why is this needed? I see everything is working just fine, but I'd really appreciate if you could explain this line to me. Thanks! 😄

Comment (GabrielaY) [reply odyssey/client/src/Profiles/UserProfile.svelte:114]: line 114

Comment (ogi02) [reply odyssey/client/src/Profiles/UserProfile.svelte:114]: You can check out [this link](https://svelte.dev/tutorial/updating-arrays-and-objects) from Svelte Docs. Basically when you set `post.isLiked = true;` this won't update references to `posts`, which is why it should be followed by `posts = posts`.

_A simple rule of thumb: the name of the updated variable must appear on the left hand side of the assignment._ - Svelte Docs

Comment (GabrielaY) [reply odyssey/client/src/Profiles/UserProfile.svelte:114]: Thanks! ❤️
